src/Internal/BalanceTx/Error.purs

```diff
@@ -4,19 +4,22 @@
 module Ctl.Internal.BalanceTx.Error
   ( Actual(Actual)
   , BalanceTxError
-      ( CouldNotConvertScriptOutputToTxInput
+      ( BalanceInsufficientError
+      , CouldNotConvertScriptOutputToTxInput
       , CouldNotGetChangeAddress
       , CouldNotGetCollateral
       , CouldNotGetUtxos
       , CollateralReturnError
       , CollateralReturnMinAdaValueCalcError
       , ExUnitsEvaluationFailed
       , InsufficientTxInputs
```

Comment:
This error is no longer thrown, but mentioned in the FAQ. I'm guessing it's replaced by `BalanceInsufficientError`?